docs/rules/no-invalid-model-keys.md

```diff
@@ -0,0 +1,108 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/no-invalid-model-keys
+description: require valid keys in model prop
+since: v3.9.0
+---
+# vue/no-invalid-model-keys
+
+> require valid keys in model prop
```

Comment:
Described as a model option in the documentation. Would you please change the wording?
https://vuejs.org/v2/api/#model

```suggestion
> require valid keys in model option
```